tutorials/quickstart/pom.xml

```diff
@@ -60,8 +60,115 @@
+          <quiet>true</quiet>
+          <!-- The maximum Java heap size to be used to run javadoc. -->
+          <maxmemory>${javadoc.maxHeapSize}</maxmemory>
+          <!-- Specifies the encoding name of the source files.-->
+          <encoding>UTF-8</encoding>
+          <!-- Set an additional parameter for the command line. -->
+          <additionalOptions>-keywords</additionalOptions>
+          <breakiterator>true</breakiterator>
+          <excludePackageNames>org.geotools.resources:org.geotools.maven:com:net.opengis:org.w3:javax:it.geosolutions</excludePackageNames>
+
+          <!-- Enables the Javadoc tool to interpret a simple, one-argument -->
+          <!-- custom block tag tagname in doc comments. Note: <placement/> -->
+          <!-- should be a combination of the letters Xaoptcmf.             -->
+          <tags>
+            <tag>
+              <name>todo</name>
+              <placement>a</placement>
+              <head>TODO:</head>
+            </tag>
+            <tag>
+              <name>todo:</name>
+              <placement>a</placement>
+              <head>TODO:</head>
+            </tag>
+            <tag>
+              <name>TODO</name>
+              <placement>a</placement>
+              <head>TODO:</head>
+            </tag>
+            <tag>
+              <name>task</name>
+              <placement>tfmc</placement>
+              <head>TODO:</head>
+            </tag>
+            <tag>
+              <name>revisit</name>
+              <placement>tfmc</placement>
+              <head>TODO:</head>
+            </tag>
+            <tag>
+              <name>generated</name>
+              <placement>Xt</placement>
+              <head>Generated</head>
+            </tag>
+            <tag>
+              <name>UML</name>
+              <placement>a</placement>
+              <head>UML:</head>
+            </tag>
+            <tag>
+              <name>uml.property</name>
+              <placement>a</placement>
+              <head>UML property:</head>
+            </tag>
+            <tag>
+              <name>generated</name>
+              <placement>X</placement>
+            </tag>
+          </tags>
+
+          <!-- Creates links to existing javadoc-generated         -->
+          <!-- documentation of external referenced classes.       -->
+          <links>
+            <link>https://docs.oracle.com/javase/11/docs/api/</link>
+            <link>https://javaee.github.io/javaee-spec/javadocs/</link>
+            <link>https://locationtech.github.io/jts/javadoc/</link>
+          </links>
+        </configuration>
+        <!-- only works with package javadoc:aggregate for use of META-INF/MANIFEST.MF Automatic-Module-Name -->
+        <reportSets>
+          <reportSet>
+            <id>aggregate</id>
+            <reports>
+              <report>aggregate</report>
+            </reports>
+            <inherited>false</inherited>
+          </reportSet>
```

Comment:
This is the part that fixes the javadoc module error message. I had to look up the fix (it does not make a lot of sense to me).

Note QuickStart is getting its own javadoc section since it is an independent project and not part of the main build.